src/Resources/views/default/edit.html.twig

```diff
@@ -15,3 +15,9 @@
         </div>
     </div>
 {% endblock content_header %}
+
+{% block body_javascript %}
+    {{ parent() }}
+
+    <script src="{{ asset('bundles/easyadminextension/js/autocomplete-create.js') }}"></script>
```

Comment:
Push it in the layout as well, at the end of block `form_entity_modal`. This will give access to the entity form modal from anywhere in the admin and avoid modifications of edit.html.twig and new.html.twig